junebug/channel.py

```diff
@@ -193,9 +193,17 @@ def _transport_config(self):
 
     @property
     def _application_config(self):
+        redis_manager = self.redis._config
+        # For tests, FAKE_REDIS cannot be an object, else deepcopy doesn't
+        # work.
+        if 'FAKE_REDIS' in redis_manager:
+            redis_manager['FAKE_REDIS'] = 'yes'
```

Comment:
Yes, came across this issue before adding those lines. We get the `deepcopy` from confmodel, and if `FAKE_REDIS` is an object instead of a string, then it fails.